pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -69,6 +69,8 @@ public class ControllerConf extends PinotConfiguration {
   public static final String CONTROLLER_MODE = "controller.mode";
   public static final String LEAD_CONTROLLER_RESOURCE_REBALANCE_STRATEGY = "controller.resource.rebalance.strategy";
   public static final String LEAD_CONTROLLER_RESOURCE_REBALANCE_DELAY_MS = "controller.resource.rebalance.delay_ms";
+  private static final String LEAD_CONTROLLER_RESOURCE_REBALANCE_NUM_THREADS =
```

Comment:
Let's keep them `public` in case we want to use these constants to set config

Reply:
This config is not for lead controller resource (lead controller resource is a Helix resource to manage lead controller). Let's rename it to `controller.executor.rebalance.numThreads`